practical/javascript/figure_labels_from_mapannotation.js

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+// All rights reserved.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+// Adds labels to image panels in OMERO.figure
+// using Key Value pairs (map annotations) on the images.
+//
+// To use, open the OMERO.figure file in your browser.
+// Select the image panels you want to add labels to.
+// Open devtools in Chrome / Firefox and select Console tab.
+// Copy the code below into the console.
+// This will load map annotations JSON for each selected image and create labels.
+// Can observe under the Network tab in devtools to see JSON loaded.
```

Comment:
That is a bit of a weird workflow.

Reply:
That is the original documentation

@will-moore: do you want to suggest something else?

Reply:
How about this? (to be added before the ```To use...``` line)
// Using the browser devtools to manipulate OMERO.figure is an experimental feature for developers.
// NB: Never paste untrusted code into your browser console.

Reply:
Sounds great.